packages/client/tests/functional/tracing/tests.ts

```diff
@@ -99,13 +99,42 @@ testMatrix.setupTestSuite(({ provider }, suiteMeta, clientMeta) => {
     })
   }
 
-  function dbQuery(statement: string): Tree {
-    return {
+  function dbQuery(statement: string, hasDriverAdapterResultSpan = true): Tree {
```

Comment:
Can we maybe just add some logic, e.g. `if starts with SELECT, hasDriverAdapterResultSpan = true`?

Reply:
Oh, `executeRaw` does not fit that :/